src/adapters/zwave/zwave-classifier.js

```diff
@@ -44,16 +44,18 @@ class ZWaveClassifier {
         }
     }
 
-    initOnOffSwitch(node, valueId) {
-        node.type = THING_TYPE_ON_OFF_SWITCH;
+    initOnOffThing(node, valueId, type) {
+        node.type = type;
         node.properties.set('on',
                             new ZWaveProperty(node, 'on', 'boolean', valueId));
```

Comment:
I don't like the fact that we're using the same property name for a sensor as a switch. And on doesn't necessarily make sense in many contexts. For example, with a door switch, does on == true mean that the door is open or that the door is closed? I can't find an example, but it's possible that a device implements both command classes, in which case using the same property name would be problematic.

The zwave documentation refers to event-triggered and event-not-triggered, so I'd be inclined to use a property name of "triggered", and still remain a boolean.